config/api/src/main/java/org/jboss/windup/config/metadata/Label.java

```diff
@@ -0,0 +1,85 @@
+package org.jboss.windup.config.metadata;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+/**
+ * @author <a href="mailto:carlosthe19916@gmail.com">Carlos Feria</a>
+ */
+public class Label
+{
+    private final String id;
+    private final String name;
+    private final String description;
+    private Set<String> supported = new HashSet<>();
+    private Set<String> unsuitable = new HashSet<>();
+    private Set<String> neutral = new HashSet<>();
+
+    private String labelString;
```

Comment:
Is `labelString` used somewhere?

Reply:
`labelString` is used in windup-web to show to the user the content of a label. This field keeps the value of the tag `<label id="id">...<label>` and all its content.

Reply:
I see, thanks for clarifying this.